### DISCUSSION TITLE
Date of the next Webinar

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#390] [Q&A] [answered]
Will there be another Webinar prior to 3/1 release of V1?  There were multiple questions we were told would be forthcoming examples-  capitation & % of billed charges. 

@shaselton-usds

**Comment by shaselton-usds** [comment, marked as answer]
The last webinar was on 3/16. In case you missed it, you can find the recording [here](https://www.cms.gov/healthplan-price-transparency/resources/webinars).
